addons/danger/functions/fnc_doSmoke.sqf

```diff
@@ -0,0 +1,72 @@
+#include "script_component.hpp"
+/*
+ * Author: nkenny
+ * Unit finds and throws smoke grenade at location
+ *
+ * Arguments:
+ * 0: Unit  <OBJECT>
+ * 1: Position <ARRAY>, optional
+ * 2: Type, corresponds to ai usage flags <string> optional
+ *
+ * Return Value:
+ * boolean
+ *
+ * Example:
+ * [bob, getpos angryJoe] call lambs_danger_fnc_doSmoke;
+ *
+ * Public: No
+*/
+
+params ["_unit", ["_pos", []], ["_type", "4 + 2"]];
+
+// no autnomous smoke throwing  <--- TODO nkenny (also do check elsewhere to preserve usefullness of this function)
+if (false) exitWith {false};
```

Comment:
never exit but still do a if?

Reply:
will fix. Its the missing space for a global CBA controlled variable :)